src/components/Advertisements/Advertisements.tsx

```
@@ -41,211 +41,269 @@
-                  loader={[...Array(6)].map((_, index) => (
-                    <div key={index} className={styles.itemCard}>
-                      <div className={styles.loadingWrapper}>
-                        <div className={styles.innerContainer}>
-                          <div
-                            className={`${styles.orgImgContainer} shimmer`}
+                {completedAdvertisements.length === 0 ? (
+                  <h4>{t('pMessage')}</h4>
+                ) : (
+                  completedAdvertisements.map((ad) => {
+                    return (
+                      <div className={styles.justifyspAdvertisements}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)